x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/DataFrame.java

```diff
@@ -94,7 +94,7 @@
     public static final Set<String> HEADER_FILTERS = new HashSet<>(
```

Comment:
@hendrikmuhs Also, this appears unused. Are there plans of using it?

Reply:
I think it's been refactored out into `ClientHelper.SECURITY_HEADER_FILTERS` (if you search for usages of that you'll see it's used in the `dataframe` plugin), but the old duplicated variable has not been removed in this file.

Reply:
OK, I'll remove it then in this PR.